Documentation/gettingstarted/k8s-install-azure-cni-steps.rst

```diff
@@ -66,7 +66,7 @@ Deploy Cilium release via Helm:
      --set global.cni.chainingMode=generic-veth \\
      --set global.cni.customConf=true \\
      --set global.nodeinit.enabled=true \\
-     --set global.azure.enabled=true \\
+     --set nodeinit.azure=true \\
```

Comment:
Do we still need to set `global.azure.enabled` also?

Reply:
Setting `global.azure.enabled=true` will change operator image to `cilium-operator-azure` as per https://github.com/cilium/cilium/blob/master/install/kubernetes/cilium/charts/operator/templates/deployment.yaml#L60. And `cilium-operator-azure` only allows `ipam.azure` mode. 

Understand that `global.azure.enabled` might cause some confusion, keen to hear your input to make it clearer as well as backward compatible.

Reply:
How about we rename  `.azure` in nodeinit to `.expect-azure-vnet` or similar to make it more clear?

And then maybe add an additional comment on the of this value the about its use-case? AFAIU, this is when we want to use azure but without using `cilium-operator-azure` that only allows `ipam.azure`.